fsspec/implementations/hdfs.py

```diff
@@ -95,94 +241,52 @@ def _open(
             **kwargs,
         )
 
-    def __reduce_ex__(self, protocol):
-        return PyArrowHDFS, self.pars
+    def checksum(self, path, env=None, **kwargs):
```

Comment:
For fsspec's purposes hash of `info` or something like that will do just fine. Let's not introduce CLI checksum here, we will stop using it ourselves in the future as well anyway, so it doesn't make sense to adopt such a complication.

Reply:
agree